### PR TITLE
Add labels for spark executors running in K8s

### PR DESCRIPTION
**Pull request by nisan1101**

### CODE REVIEW

File: service_configuration_lib/spark_config.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
     'spark.kubernetes.authenticate.clientKeyFile',
     'spark.kubernetes.authenticate.clientCertFile',
     'spark.kubernetes.container.image.pullPolicy',
+    'spark.kubernetes.executor.label.yelp.com/paasta_service',
+    'spark.kubernetes.executor.label.yelp.com/paasta_instance',
```
Comment (solarkennedy) [inline service_configuration_lib/spark_config.py:30]: Can you also add cluster?

https://github.com/Yelp/paasta/blob/5d3523b5ec57c9857137a7cb6cecebf9cd0931f7/paasta_tools/setup_kubernetes_cr.py#L265

**Comment by solarkennedy** [comment]
I think I'm wrong about these labels?
https://github.com/Yelp/paasta/pull/2631/files

@keymone do we need both? or just paasta.yelp.com/*

**Comment by keymone** [comment]
`paasta.yelp.com/name` is the correct scheme, `yelp.com/paasta_` is deprecated

**Comment by nisan1101** [comment]
@solarkennedy @keymone  Im adding the labels to satisfy https://github.com/Yelp/synapse-tools/blob/master/src/synapse_tools/generate_container_ip_map.py#L71. Should I add both labels?

**Comment by keymone** [comment]
no need to add both, just the new scheme

**Pull request merged**